scripts/benchmark/benchmark_requests.py

```diff
@@ -139,6 +160,7 @@
         f.write("status,time\n")
         f.writelines(f"{s},{t}\n" for s, t in zip(success, times, strict=False))
 
+    print(f"Results saved to the following directory: {outfile}")
```

Comment:
`outfile` is a file, not directory. So update the string to: `Results saved to:`